src/api/test-use.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import API from './auth-api';
+import { User } from '../types';
+
+const TestAPI = (): JSX.Element => {
+  const [id, setId] = useState<unknown>(null);
```

Comment:
❗️
Мы знаем, что придёт, а также сами задаём базовое значение, поэтому логично переписать так:
```suggestion
  const [id, setId] = useState<number | null>(null);
```